force-app/main/default/layouts/ServiceSession__c-Service Session Layout.layout-meta.xml

```diff
@@ -111,7 +110,7 @@
     <showRunAssignmentRulesCheckbox>false</showRunAssignmentRulesCheckbox>
     <showSubmitAndAttachButton>false</showSubmitAndAttachButton>
     <summaryLayout>
-        <masterLabel>00h21000001j3nm</masterLabel>
+        <masterLabel>00h1g000001bXv6</masterLabel>
```

Comment:
we don't have to commit these changes, but nor do they hurt.

Reply:
@bethbrains , I usually try to. Glad to know we don't have to do it. Will keep in mind for the next one